aidants_connect_habilitation/views.py

```diff
@@ -136,8 +147,25 @@ def get_success_url(self):
             kwargs={"issuer_id": self.saved_model.issuer_id},
         )
 
-    def get_context_data(self, **kwargs):
-        return {**super().get_context_data(**kwargs), "step": self.step}
+    def send_mail(self, email: str):
```

Comment:
Donc cette méthode-là envoie le mail de rappel de l'adresse demandeur, mais pas la confirmation. Ça pourrait induire en erreur plus tard lors de la maintenance du code, je propose de renommer la méthode : 

```suggestion
    def send_issuer_profile_reminder_mail(self, email: str):
```